### DISCUSSION TITLE
Is there any plan for MultiLanguage support for entities?

### DISCUSSION THREAD

**Discussion by ali-gol** [abpframework/abp#10383] [Ideas] [answered]
Hi,

Multi language feature for entities are usually a part of business logic but maybe ABP framework can offer an infrastructure for this solution. Is there any plan for this in the future? 

I want to give an example in case of the requirement is not clear enough:
For example, i want to build a simple cms application for myself but at the same time i want so support multi language in my website. In order to do that, i need to store different languages of my articles.
Lets say my article has id, title and description. Those title and description needs to be stored in seperate table with the LanguageId foreign key (at least that would be the way i follow). 
This is the requirement.

Thanks

**Comment by realLiangshiwei** [comment, marked as answer]
Hi,

We tried before, but it is not suitable for the current frame design. see https://github.com/abpframework/abp/issues/1754#issuecomment-826184860
